Autopilot/Src/altimeterThreadInterface.cpp

```diff
@@ -1,12 +1,12 @@
-#include "altimeterThreadInterface.h"
-#include "imu.hpp"
+#include "IMUThreadInterface.h"
+ #include "altimeter.hpp"
```

Comment:
huh? what's happening here

Reply:
shouldn't it be `altimeterThreadInterface.h`?

Reply:
hmm something weird's happening with my commits let me fix them up rq

Reply:
ok so I made a force push because I reverted to a previous commit, and it seems like in files changed everything is how I want it to be. The Sensorfusion is in its own library, but despite that Travis still doesn't build. Would you know why?